resolwe/flow/tests/test_commands.py

```diff
@@ -88,3 +91,53 @@ def test_process_register_no_admin(self):
         err = StringIO()
         self.assertRaises(SystemExit, call_command, 'register', path=[PROCESSES_DIR], stderr=err)
         self.assertEqual('Admin does not exist: create a superuser\n', err.getvalue())
+
+
+class ListDockerImagesTest(ProcessTestCase):
```

Comment:
Why does this require `ProcessTestCase`? The above test cases only call `register` via `call_command`, I think this should be enough here as well?

The process test case is usually used for test cases where processes need to be executed, but here I think it should be enough for them to just be registered.